lib/core/src/lib/form/models/widget-visibility.model.ts

```diff
@@ -86,7 +86,7 @@ export class WidgetVisibilityModel {
     }
 
     get rightValue(): any {
-        if (this.json.rightValue) {
+        if (this.json.rightValue != null) {
```

Comment:
shouldn't we use here strict (`!==`) checking? with this implementation, if rightValue is undefind, the condition will be met, is this intended?

Reply:
It was intended but not sure whether the best solution